website_multi_theme/models/website.py

```diff
@@ -82,18 +85,15 @@ def _duplicate_view_for_website(self, pattern, xmlid, override_key):
             return result
         # Copy patterns only for current website
         key = xmlid if override_key else pattern.key
+
         result = pattern.copy({
             "active": pattern.was_active,
             "arch_fs": False,
-            "customize_show": False,
```

Comment:
@Yajo is there any reason why this was set to False? Do you remember the case when it's required?